test/AlgebraicGeometry/ToricVarieties/toric_schemes.jl

```diff
@@ -167,6 +167,8 @@ end
   x1, x2, x3 = gens(cox_ring(p231)) 
   my_ideal = ideal([x1*x2]) 
   II = ideal_sheaf(p231, my_ideal) 
+  JJ = Oscar.ToricIdealSheafFromCoxRingIdeal(p231, my_ideal)
+  @test II == JJ
```

Comment:
Do we want to export this function (or its `snake_case` equivalent, see above)? If so, the `Oscar.` can be removed.

Reply:
(This also applies to the 2nd test that you are adding.)